packages/x-date-pickers-pro/src/tests/describeValidation.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-env mocha */
+import { BaseDateValidationProps } from '@mui/x-date-pickers/internals';
+// import testDayViewValidation from './testValidation/testDayViewValidation';
+import testTextFieldValidation from './testValidation/testTextFieldValidation';
+import { DayRangeValidationProps } from '../internal/models';
+
+type ValidationProps =
+  | keyof (DayRangeValidationProps<any> & BaseDateValidationProps<any>)
+  | 'minDateTime'
+  | 'maxDateTime';
+
+const defaultAvailableProps: ValidationProps[] = [
+  // from now
+  'disablePast',
+  'disableFuture',
+  // date range
+  'minDate',
+  'maxDate',
+  // time range
+  'minDate',
+  'maxDate',
+  // date time range
+  'minDateTime',
+  'maxDateTime',
+  // specific moment
+  'shouldDisableDate',
+];
+
+type AvailableTests = 'day' | 'textField';
+
+const testsToRun = {
+  // day: testDayViewValidation,
+  textField: testTextFieldValidation,
+};
+
+interface ConformanceOptions {
+  render: any;
+  clock: any;
+  after?: () => void;
```

Comment:
I don't seea ny usage of this callback, including in the community version, what would be the use case for it ?

Reply:
No idea, I assume if the core team added such option it was for good reason but Yes for now we don't need it

Reply:
Oh you just took it from `describeConformance` ? OK to keep the same options for now even  if not used